simplematrixbotlib/match.py

```diff
@@ -99,18 +108,17 @@ def command(self, command=None):
             Returns the string after the prefix and before the first space if no arg is passed to this method.
         """
 
-        if self._prefix == self.event.body[0:len(self._prefix)]:
-            body_without_prefix = self.event.body[len(self._prefix):]
-        else:
-            body_without_prefix = self.event.body
-
-        if not body_without_prefix:
-            return []
+        # we cache this part
+        if self._body_without_prefix is None:
+            if self._prefix == self.event.body[0:len(self._prefix)]:
+                self._body_without_prefix = self.event.body[len(self._prefix):]
+            elif not self.mention():  # if mention() is True then it also sets the _body_without_prefix
+                self._body_without_prefix = self.event.body
 
```

Comment:
found a new issue:
trying to get command with mention but no prefix always matches prefix

possibilities:
- should mention have priority over prefix? => delete a mention if it exists before the using the whole body
- if there is no prefix, then disable mention

Reply:
@KrazyKirby99999 I have implemented the first variant now (3a5d621). Please comment if you disagree or mark as resolved otherwise.